zerver/openapi/zulip.yaml

```diff
@@ -5450,6 +5450,86 @@ paths:
                         "result": "success",
                         "msg": "",
                       }
+  /users/{email}:
+    get:
+      operationId: get_user_by_email
+      tags: ["users"]
+      description: |
+        Fetch details for a single user in the organization.
+
+        `GET {{ api_url }}/v1/users/{email}`
+
+        You can also fetch details on [all users in the organization](/api/get-users).
+
+        *This endpoint is new in Zulip Server 3.0 (feature level 1).*
```

Comment:
This statement is inaccurate -- we need to allocate a new feature level as this will be new as of when we merged it, as part of Zulip 4.0.